.github/workflows/Publish.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8, 3.9]
-        os: [windows-2019, macOS-10.15, macOS-11.0]
+        os: [windows-2019]
```

Comment:
Where is ubuntu-20.04 and 18.04?

Reply:
For publishing linux is handled by a custom image / action for manylinux2010. 
The action is in PublishLinux.yml and the detailed repo is [here](https://github.com/allen-cell-animated/python-wheels-manylinux-build)

Reply:
Gotcha. Thanks!